### ISSUE TITLE
[ST2] Package Control fails to install dependencies on Linux

### ISSUE THREAD

**Issue by deathaxe** [wbond/package_control#1213]
### Description

For testing purposes I installed a clean ST2 on a Linux VM and added package control by executing the [Install Script](https://packagecontrol.io/installation#st2). After some restarts system runs ok.

Then I added my GitGutter repo to the package folder and restarted ST2. 
Package control starts to install the 2 dependencies `markdupsafe` and `python-jinja2` with the attached result below. 

I guess curl_downloader.py tries to decode the result code 200 which fails due to prepending 'HTTP/2'?

### Information

- ST2 built #2221
- Kali Linux x64

### Console output

```python
Package Control: Installing 2 missing dependencies
Exception in thread Thread-4:
Traceback (most recent call last):
  File ".\threading.py", line 532, in __bootstrap_inner
  File "./package_control/automatic_upgrader.py", line 111, in run
  File "./package_control/automatic_upgrader.py", line 143, in install_missing
  File "./package_control/package_manager.py", line 1106, in install_package
  File "./package_control/download_manager.py", line 340, in fetch
  File "./package_control/downloaders/curl_downloader.py", line 162, in download
ValueError: invalid literal for int() with base 10: 'HTTP/2 200 '
```

**Comment by deathaxe** [comment]
This issue seems dedicated to Kali Linux for some reason. All dependencies are properly installed with Linux Mint 18.

**Comment by wbond** [comment]
It sounds like Kali Linux has a version of curl with http2 support. It sounds as if there is a bug with parsing the output from curl.

**Comment by deathaxe** [comment]
`curl --help` lists an `--http2` option, so you seem right.

**Comment by wbond** [comment]
This should be fixed by 341553cfb6dcf5138ae17b24a9eca20c6dd008b1
